### PR TITLE
Add social gallery section

### LINKED ISSUE #1466: Add social gallery section
Implement social gallery section according to [mockup](https://www.figma.com/file/jRG4EyQXRDMGfyLqut08KP/iFixit---Working-files?node-id=10434%3A211429&t=O9f1o3dzOVlT1Rj2-4)

Screenshot 2023-03-14 at 18 32 00

### PR DESCRIPTION
**Pull request by dhmacs**
closes #1466 

❗❗❗After this PR is merged Strapi should be redeployed in production ❗❗❗

cc @sterlinghirsh @danielbeardsley @masonmcelvain 

## QA

1. Open a [product page preview](https://react-commerce-git-add-social-gallery-section-ifixit.vercel.app/store)
2. Verify that the social gallery is implemented as specified in #1466 
3. Verify that the section is responsive

> 👁️ Data is editable from Strapi

### CODE REVIEW

**Review by masonmcelvain — APPROVED** [review]
CR ✌🏻 Looks good! deploy_block ✌🏻 on the permissions thing, I don't want to accidentally break CI again. If the permissions don't apply in this case, feel free to un_deploy_block and merge @dhmacs

File: backend/src/api/social-post/controllers/social-post.ts
[the remark this answers is not in the document]
Comment (dhmacs) [reply backend/src/api/social-post/controllers/social-post.ts:1]: Gosh, it's incredible how I forget about it all the time 🤦‍♂️ I'm gonna fix it now

**Review by masonmcelvain — APPROVED** [review]
CR ✌🏻 un_deploy_block ✌🏻 Glad we caught it this time! 😅

**Comment by deltuh-vee** [comment]
dev_block ✊ 
I'm not seeing the social gallery appear anymore.
image

**Comment by k0rvusk0r4x** [comment]
The social gallery is still there, still looks good, and is still responsive.

QA 😺

**Comment by dhmacs** [comment]
@masonmcelvain reminder to redeploy Strapi in production ❗

**Comment by masonmcelvain** [comment]
Redeployed ✅

**Pull request merged**